src/compute_pressure.cpp

```diff
@@ -152,6 +186,11 @@ void ComputePressure::init()
   if (nvirial) {
     vptr = new double*[nvirial];
     nvirial = 0;
+    if (hybridpairflag && force->pair) {
+      PairHybrid *ph = (PairHybrid *) force->pair;
+      ph->no_virial_fdotr_compute = 1;
```

Comment:
this seems problematic, what if this compute goes away (uncompute), 
then the setting in PH will not be un-done
better way to do this might be, PH in its init_style() sets this no_virial
flag in its usual way (currently in flags()), but also loops over computes looking to see
if any (like this one) sets a flag that tells it to override the setting
it would normally make

Reply:
> this seems problematic, what if this compute goes away (uncompute),
> then the setting in PH will not be un-done

i don't think this would be too drastic, since this use case would be rather unusual.

> better way to do this might be, PH in its init_style() sets this no_virial
> flag in its usual way (currently in flags()), but also loops over computes looking to see
> if any (like this one) sets a flag that tells it to override the setting
> it would normally make

this seems to me like a big hammer for a small problem. after all, the change doesn't break functionality of the affected pair style.

Reply:
but it makes PH compute something (the virial) inefficiently,
so I think its bad to not remove that requirement if the compute goes away